README.md

```diff
@@ -268,6 +269,7 @@ Run it as a sub-process from your favorite language; `pip install mintapi` creat
       --limit               Number of records to include from the API.  Default is 5000.
       --show-pending        Retrieve pending transactions.
                             Used with --transactions
+      --fail-if-stale       At login, Mint attempts to refresh your data.  If you wish to exit when the fail syncs, use this option.
```

Comment:
Should this say "when the sync fails"? Or, am I just reading it poorly?

Reply:
Yup, I thought I caught all of those instances.